xl/trax/track.py

```diff
@@ -222,7 +222,7 @@ def __init__(self, uri=None, scan=True, _unpickles=None):
                 state. not for normal use.
         """
         # don't re-init if its a reused track. see __new__
-        if self._init == False:
+        if not self._init:
```

Comment:
I like using `is False` in some instances like this one only because it's more readable.